cmd/node/config/gasSchedules/gasScheduleV7.toml

```diff
@@ -41,6 +41,7 @@
     DelegateVote          = 50000000
     RevokeVote            = 50000000
     CloseProposal         = 50000000
+    ClearProposal         = 50000000
```

Comment:
when you will merge the feat branch, please take into account that we have also the gasScheduleV8.toml

Reply:
👍